src/routes/(console)/project-[region]-[project]/databases/database-[database]/table-[table]/export/+page.svelte

```diff
@@ -0,0 +1,327 @@
+<script lang="ts">
+    import { onMount, getContext } from 'svelte';
+    import { base } from '$app/paths';
+    import { page } from '$app/state';
+    import { goto } from '$app/navigation';
+    import { Wizard } from '$lib/layout';
+    import {
+        Fieldset,
+        Layout,
+        Typography,
+        Icon,
+        Divider,
+        Tooltip,
+        Skeleton
+    } from '@appwrite.io/pink-svelte';
+    import { Button, InputText, InputSelect, InputCheckbox, Form } from '$lib/elements/forms';
+    import type { FormContext } from '$lib/elements/forms/form.svelte';
+    import { addNotification } from '$lib/stores/notifications';
+    import { sdk } from '$lib/stores/sdk';
+    import { IconInfo } from '@appwrite.io/pink-icons-svelte';
+    import type { Models } from '@appwrite.io/console';
+    import { table } from '../store';
+    import { queries, type TagValue } from '$lib/components/filters/store';
+    import { TagList } from '$lib/components/filters';
+    import { Submit, trackEvent, trackError } from '$lib/actions/analytics';
+    import { toLocalDateTimeISO } from '$lib/helpers/date';
+
+    let showExitModal = $state(false);
+    let formComponent: Form;
+    let isSubmitting = $derived(getContext<FormContext>('form')?.isSubmitting);
```

Comment:
should be `let isSubmitting = writable(false);`

Reply:
The form component already handles this, we just need to `bind` it to our external var to get the updates like - `bind:isSubmitting`.